etl/steps/data/garden/worldbank_wdi/2024-05-20/wdi.py

```diff
@@ -84,18 +82,12 @@ def mk_omms(table: Table) -> Table:
 
     # for convenience, uses the same generic "Our World in Data based on..."
     # source for all OMMs created here
-    source = deepcopy(table["sp_pop_totl"].metadata.sources[0])
-    omm_source = Source(
-        name="Our World in Data based on World Bank",
-        description=None,
-        url=source.url,
-        source_data_url=source.source_data_url,
-        owid_data_url=source.owid_data_url,
-        date_accessed=source.date_accessed,
-        publication_date=source.publication_date,
-        publication_year=source.publication_year,
-        published_by=f"Our World in Data based on {source.published_by}",
-    )
+    omm_origin = table["sp_pop_totl"].metadata.origins[0].copy()
+
+    # rewrite "World Development Indicators" to "Our World in Data based on World Bank"
+    omm_origin.title = "Our World in Data based on World Bank"
```

Comment:
Why this? The title of the data product should be "World Development Indicators". Also, we shouldn't use "Our World in Data based on..." anymore.

Reply:
And now that we are modifying the origin of each individual indicator, why not editing the full citation too, to be consistent? We could simply prepend the `producer` to the current full citation.

Reply:
Good idea, thanks!